app/models/shelter.rb

```diff
@@ -7,4 +7,15 @@ def pet_count
     pets.count
   end
 
+  def self.order_by_num_adopatable
+    left_outer_joins(:pets)
+    .where(:pets => {:adoption_status => true})
+    .group(:id)
+    .order('COUNT(shelter_id) DESC')
```

Comment:
This is the only way I could figure out how to order by the number of adoptable pets. I am curious as to whether there is a better way to do this or if there is a way to keep shelters in the table if they have no adoptable pets.